types/src/on_chain_config/aptos_features.rs

```diff
@@ -53,6 +53,7 @@ pub enum FeatureFlag {
     BN254_STRUCTURES = 43,
     WEBAUTHN_SIGNATURE = 44,
     RECONFIGURE_WITH_DKG = 45,
+    JWK_CONSENSUS = 46,
```

Comment:
I remember vtxn is enabled by consensus config and makes it more complicated. Any reason to do so instead of adding a flag here?

Reply:
Because that piece is "whether consensus should pull vtxn", so consensus behavior specific?

Reply:
Do we plan to add limits for vtxn? That will force us to use the consensus config. Otherwise I think flag is easier to keep track of.

Reply:
i thought vtxn limits will be node config, similar to [these](https://github.com/aptos-labs/aptos-core/blob/cc677d7f2d9a3cb118625158e419b1d142c9a8b5/config/src/config/consensus_config.rs#L22-L30)?

Reply:
@grao1991 to confirm: is adding vtxn sending/receiving limits as simple as adding them as fields of [this struct](https://github.com/aptos-labs/aptos-core/blob/cc677d7f2d9a3cb118625158e419b1d142c9a8b5/config/src/config/consensus_config.rs#L18), and provide reasonable default value (e.g., half of existing limits for the entire block)

Reply:
I agree with putting the limits on chain. This should not be a local config that operators can change.

Reply:
fixed